README.md

```diff
@@ -38,9 +46,9 @@ provide [a script](./albert/AWS_runner.ipynb) for launching experiments using pr
 ## Acknowledgements
 
 This project is the result of a collaboration between
-[Yandex](https://research.yandex.com/), [Hugging Face](https://huggingface.co/), [MIPT](https://mipt.ru/english/)
-, [HSE University](https://www.hse.ru/en/), [University of Toronto](https://www.utoronto.ca/)
-, [Vector Institute](https://vectorinstitute.ai/) and [Neuropark](https://neuropark.co/).
```

Comment:
This was displayed as "MIPT , HSE University, University of Toronto , Vector Institute"